schedule/schedule_commands.go

```diff
@@ -672,7 +672,7 @@ func encodeSearchAttributes(sa map[string]interface{}) (*commonpb.SearchAttribut
 	if len(sa) == 0 {
 		return nil, nil
 	}
-	dc := dataconverter.DefaultDataConverter()
+	dc := converter.GetDefaultDataConverter()
```

Comment:
Correctness: only used in `getMemoAndSearchAttributesForSchedule`, therefore correct by [above](https://github.com/temporalio/cli/pull/384/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#r1406077577).